web/src/metarpheus/model-ts.ts

```diff
@@ -1,23 +1,42 @@
-
-/* tslint:disable */
-
+// DO NOT EDIT MANUALLY - metarpheus-generated
+import * as t from 'io-ts'
 
 export interface AccessToken {
   value: string,
   expiresAt: string
 }
 
+export const AccessToken = t.interface({
```

Comment:
I suggest we use [`t.strict`](https://github.com/gcanti/io-ts#strictexact-interfaces) instead of `t.interface`

Reply:
Is there a config param to pass to `metarpheus` for that? This file is auto-generated

Reply:
ups, sorry then 👍